test/commands/translation/push.test.ts

```diff
@@ -48,14 +48,14 @@ describe("commands/translation/push", () => {
           sinon.match(({ flags }) => {
             return isEqual(flags, {
               "service-token": "valid-token",
-
               environment: "development",
             });
           }),
           sinon.match({
             content: '{"welcome":"hello!"}',
             locale_code: "en",
             namespace: "admin",
+            format: "json",
```

Comment:
Is it worth adding tests for the new behaviors/functionalities we just added? There are some bits in this PR that look curious to me, and preferably we include tests to verify the new stuff we are adding is working as intended.

Reply:
Yeah, I think the main thing is whether on validate/push it correctly falls back between the different file formats, so I'll add cases where we don't specify the format and only have a po file and make sure validate and get po back